### DISCUSSION TITLE
CustomWebView -v12

### DISCUSSION THREAD

**Discussion by vknow360** [vknow360/CustomWebView#25] [General]
**New Blocks**
![image](https://user-images.githubusercontent.com/41724811/223966700-e6fe4ee5-06f6-47b2-a3f8-f522cc23e829.png)

**Changes**
- Added support for `localhost` urls
- Major changes and added new blocks in `DownloadHelper`

**Bug Fixes**
- Some minor and major bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/vknow360/CustomWebView/releases/tag/v12'>CustomWebView -v12</a>.</em>

**Comment by alanscot** [comment]
So now I have accumulated :-
size
 77kb				com.sunny.CustomWebView (1).aix"
 15kb				com.sunny.CustomWebView (13).aix"
 84kb				com.sunny.CustomWebView.aix"
 77kb				com.sunny.CustomWebView_V12.aix"
 68kb				com.sunny.CustomWebView-Kodular.aix"
What does the 84kb version have that v12 doesn't ? (84kb version Downloaded Jan 15th 2023)
And should v12 be the same size as a version I downloaded in January 3rd? ( 77kb)

**Reply by vknow360** [reply]
Hi, please download latest version from releases.
https://github.com/vknow360/CustomWebView/releases

**Comment by QualopGames** [comment]
Hello sir i have made all necessary changes and updated to the main .java webviewer file in order to update it by make a new .aix extension if you want. Updated based the file i will give you.
[CustomWebView.java](https://github.com/user-attachments/files/26462026/CustomWebView.java)
[BrowserPromptHelper.java](https://github.com/user-attachments/files/26462027/BrowserPromptHelper.java)
[DownloadHelper.java](https://github.com/user-attachments/files/26462028/DownloadHelper.java)
[WView.java](https://github.com/user-attachments/files/26462029/WView.java)
Check them a bit, but i think they are better for android 16+ and way more functional. Please make the update of extension.
